kalliope/neurons/volume/volume.py

```diff
@@ -0,0 +1,83 @@
+import logging
+
+from kalliope.core import NeuronModule
+import alsaaudio
+
+from kalliope.core.NeuronModule import InvalidParameterException
+
+logging.basicConfig()
+logger = logging.getLogger("kalliope")
+
+
+class SoundManager(object):
+
+    try:
+        m = alsaaudio.Mixer()
+    except alsaaudio.ALSAAudioError:
+        # no master, we are on a Rpi
+        try:
+            m = alsaaudio.Mixer("PCM")
+        except alsaaudio.ALSAAudioError:
+            # no audio config at all
+            m = None
+
+    @classmethod
+    def set_volume(cls, volume_level):
+        if cls.m is not None:
+            cls.m.setvolume(int(volume_level))
+
+    @classmethod
+    def get_volume(cls):
+        if cls.m is not None:
+            vol = cls.m.getvolume()
+            return int(vol[0])
+        return None
+
+
+class Volume(NeuronModule):
+
+    def __init__(self, **kwargs):
+        super(Volume, self).__init__(**kwargs)
+
+        self.level = kwargs.get('level', None)
```

Comment:
Level expects an `int` and the default value is`None`. it can lead to type incompatibility in the next linesm such as `self.level - current_level`. 
Maybe we could replace the default value to `0`

Reply:
Sorry i merged before answering. I saw that you approved.
Anyway, the level is tested in the method below. It will fail if it's not an integer. And this parameter is mandatory.